src/components/ProductRecommendations/index.jsx

```diff
@@ -0,0 +1,109 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Kvass from '@kvass.ai/core-sdk';
+import T from 'src/utils/i18n';
+import ProductImage from 'src/components/ProductImage';
+import Spinner from 'src/components/Spinner';
+import Actions from 'src/reflux/Actions';
+import utils from 'src/utils';
+
+class ProductRecommendations extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: false,
+      recommendations: [],
+      visibleRecIndex: 3, // endIndex of what we are showing from array
+      size: 9 // no pagination on recommendations
+    };
+    this.kvass = new Kvass();
+  }
+
+  componentDidMount() {
+    const body = {model_type: this.props.modelType, source_id: this.props.sourceId, source: this.props.source, destination: this.props.destination, size: this.state.size};
+    this.setState({isLoading: true});
+    this.kvass.aiModel().getRecommendations({body, urlParams: {expand: 'response.object'}}, (error, recommendations, res) => {
+      if (error) {
+        this.setState({isLoading: false, recommendations: []});
+        return;
+      }
+      this.setState({isLoading: false, recommendations: utils.parseRecommendations(recommendations)});
```

Comment:
We will need to stop doing that, as we talked earlier...

Reply:
Yeah, created an issue for it. #75